sphinx_gallery/scrapers.py

```diff
@@ -47,6 +49,22 @@ def _import_matplotlib():
     return matplotlib, plt
 
 
+def _matplotlib_fig_titles(fig):
+    titles = []
+    # get supertitle if exists
+    if fig._suptitle:
+        titles.append(fig._suptitle.get_text())
```

Comment:
Since this is private API, it might be safe to do:

```suggestion
    suptitle = getattr(fig, "_suptitle", None)
    if suptitle is not None:
        titles.append(suptitle.get_text())
```

Reply:
thanks! I couldn't find another way to get the suptitle.